ios/PurchasesHybridCommon/PurchasesHybridCommonSwift/CommonFunctionality.swift

```diff
@@ -0,0 +1,505 @@
+//
+//  CommonFunctionality.swift
+//  PurchasesHybridCommonSwift
+//
+//  Created by Andrés Boedo on 4/19/22.
+//  Copyright © 2022 RevenueCat. All rights reserved.
+//
+
+import Foundation
+import StoreKit
+import Purchases
+
+
+// todo: rename back to RCCommonFunctionality
+@objc(RCCommonFunctionality) public class CommonFunctionality: NSObject {
+
+    // MARK: properties and configuration
```

Comment:
this is the big one. I organized stuff a bit more into sections so it's easier to locate things now